src/Explorer/Panes/DeleteCollectionConfirmationPaneComponent.tsx

```diff
@@ -0,0 +1,201 @@
+import * as NotificationConsoleUtils from "../../Utils/NotificationConsoleUtils";
```

Comment:
Meta comment: I think we should stop suffixing all of our component files with "Component". Most of the things we work on are components. I don't think its providing value.

Reply:
Agreed. I'll do PR to rename all the react components.